redisdb/datadog_checks/redisdb/data/conf.yaml.example

```diff
@@ -36,6 +36,12 @@ instances:
     #
     # password: <PASSWORD>
 
+    ## @param enable_client_command - boolean - optional
+    ## Enable to collect metrics using the `CLIENT` command
+    ## This requires the Redis CLIENT command to be available on your servers.
+    #
+    # enable_client_command: false
```

Comment:
`collect_client_metrics` might be more aligned with existing pattern used in other integrations `collect_something: true/false`